.ci-support/production-setup.sh

```diff
@@ -15,7 +15,7 @@ if [ -n "${PRODUCTION_CHANGE_BRANCH}" ]; then
 else
     echo "No updates to production branch (${CURRENT_BRANCH})"
 fi
-CURRENT_FORK_OWNER=""
+CURRENT_FORK_OWNER="illinois-ceesd"
```

Comment:
👍

Reply:
Yeah, this really should be down inside the conditional there:

`if [ -z "${CURRENT_FORK_OWNER}" ]; then CURRENT_FORK_OWNER="illinois-ceesd"`

or similar.